AmericanMedicalAssociation/AMA-style-guide
# EWL 3264 - Category Highlight with Image

Ticket: https://issues.ama-assn.org/browse/EWL-3264

To test:
- Go to Molecule > Components > Category Highlight with Image
- Check component at all viewports
- Compare to PSD
